Tests/OAuth/ResourceOwner/BufferAppResourceOwnerTest.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/*
+ * This file is part of the HWIOAuthBundle package.
+ *
+ * (c) Hardware.Info <opensource@hardware.info>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace HWI\Bundle\OAuthBundle\Tests\OAuth\ResourceOwner;
+
+use HWI\Bundle\OAuthBundle\OAuth\ResourceOwner\BufferAppResourceOwner;
+
+class BufferAppResourceOwnerTest extends GenericOAuth2ResourceOwnerTest
+{
+    protected $userResponse = <<<json
+{
+    "id": "4f0c0a06512f7ef214000000"
```

Comment:
@Soullivaneuh yes, to this scenario.

See more in official documentation: https://buffer.com/developers/api/user